src/lib/swagger-express-ts/swagger.builder.ts

```diff
@@ -195,7 +196,14 @@ export function build(buildDefinition: ISwaggerBuildDefinition): void {
     );
   }
   if (buildDefinition.models) {
-    SwaggerService.getInstance().setDefinitions(buildDefinition.models);
+    const models: any = {};
+    _.forOwn(buildDefinition.models, (value, key) => {
+      models[key] = {
+        name: key,
+        definition: value
+      };
+    });
+    SwaggerService.getInstance().setDefinitions(models);
```

Comment:
I don't know if possible use  setDefinitions from "external"  but i've change a little bit the structure, so it make a breaking change